### PR TITLE
Add images for Python 3.10

### PR DESCRIPTION
**Pull request by olivierbellone**
This PR:
- bumps uvicorn from 0.15.0 to 0.17.6
- add normal, slim and Alpine images for Python 3.10
- remove images for Python 3.6

It was necessary to bump uvicorn for the 3.10 images to build successfully (I think due to the dependency on httptools being bumped in that version). uvicorn dropped support for Python 3.6 in 0.17.0 so I had to remove the 3.6 images. 3.6 is at EOL anyway so I think this is the right call.

@tiangolo Any chance this could be merged soon? Once it's merged I can take care of updating the starlette and fastapi repos too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,18 @@
 
 ## Supported tags and respective `Dockerfile` links
 
-* [`python3.9`, `latest` _(Dockerfile)_](https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/python3.9.dockerfile)
+* [`python3.10`, `latest` _(Dockerfile)_](https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/python3.10.dockerfile)
+* [`python3.9`, _(Dockerfile)_](https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/python3.9.dockerfile)
 * [`python3.8`, _(Dockerfile)_](https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/python3.8.dockerfile)
 * [`python3.7`, _(Dockerfile)_](https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/python3.7.dockerfile)
 * [`python3.6` _(Dockerfile)_](https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/python3.6.dockerfile)
```
Comment (vitor-amartins) [inline README.md:9]: Maybe this should be remove too?

Comment (olivierbellone) [reply README.md]: Good call. I removed it, as well as all other remaining references to 3.6.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -11,15 +11,17 @@ jobs:
       matrix:
         image:
           - name: latest
-            python_version: "3.9"
+            python_version: "3.10"
+          - name: latest
```
[the remark this answers is not in the document]
Comment (olivierbellone) [reply .github/workflows/test.yml]: Good eye!

**Review by vitor-amartins — APPROVED** [review]

**Review by BoYanZh — APPROVED** [review]

**Comment by vitor-amartins** [comment]
@tiangolo any chances of this being merged soon?

**Comment by azhard4int** [comment]
@tiangolo could this be merged, we would like to use Python 3.10 at our current organization.

**Comment by vmesel** [comment]
@tiangolo can you help us here? I'm another user looking for this image.

**Comment by dingdayu** [comment]
@tiangolo Maybe you missed the news here, please give some comments.

**Review by gaelgoth — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks @olivierbellone! :bow: :coffee: 

This was handled in this PR: https://github.com/tiangolo/uvicorn-gunicorn-docker/pull/99

I prefer to make the changes isolated and release/push by parts. I didn't want to remove Python 3.6 in the same PR adding 3.10 and also the Uvicorn upgrade, etc.

As this is now handled, I'll close this PR, but thanks for all the effort! :rocket:

**Pull request closed without merging**